packages/css-library/src/tokens/measure.scss

```diff
@@ -0,0 +1,8 @@
+$tokens-measure: (
```

Comment:
I wonder if this name should be a little more descriptive.

Reply:
Nevermind, I see now that USWDS also uses this terminology. I don't care for it, but probably better to stay in sync with them on this.